shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1344,15 +1365,15 @@ - (void)showTextInput {
   if (!_enableFlutterTextInputViewAccessibilityTimer) {
     _enableFlutterTextInputViewAccessibilityTimer =
         [[NSTimer scheduledTimerWithTimeInterval:kUITextInputAccessibilityEnablingDelaySeconds
-                                          target:self
-                                        selector:@selector(enableActiveViewAccessibility:)
+                                          target:[FlutterTimingProxy proxyWithTarget:self]
```

Comment:
This has the potential to create another crash.  The NSTimer has a retain on the FlutterTimingProxy, but the FlutterTimingProxy doesn't have a retain on the FlutterTextInputPlugin.  There isn't anything that clears out the NSTimer when the FlutterTextInputPlugin is deleted.

Reply:
@gaaclarke Actually, we have, please see https://github.com/flutter/engine/blob/a0b52ae6bf315977827f9a6743e5feef479a1aa5/shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm#L1268.